app/src/test/java/org/simple/clinic/home/HomeScreenViewEffectHandlerTest.kt

```diff
@@ -0,0 +1,46 @@
+package org.simple.clinic.home
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.verifyNoMoreInteractions
+import io.reactivex.Observable
+import org.junit.Test
+import org.simple.clinic.facility.FacilityConfig
+import org.simple.clinic.mobius.EffectHandlerTestCase
+import org.simple.clinic.patient.PatientRepository
+import org.simple.clinic.util.scheduler.TestSchedulersProvider
+import org.simple.sharedTestCode.TestData
+import java.util.UUID
+
+class HomeScreenViewEffectHandlerTest {
```

Comment:
Let's rename this to `HomeScreenEffectHandlerTest`. View effects are part of effect handler tests